client/src/locales/en-US.json

```diff
@@ -491,7 +492,8 @@
                 "allowed": "Enabled",
                 "forbidden": "Disabled",
                 "title": "Configuration",
-                "serverAddr": "Server address"
+                "serverAddr": "Server address",
+                "copyPath": "Copy"
```

Comment:
Displayed when the button has been clicked, so it should be `Copied` or `Copied!`